infrastructure/cdkv1/lokafold-app/app.py

```diff
@@ -0,0 +1,115 @@
+import os
+import sys
+import logging
+import boto3
+from chalice import Chalice
+from chalicelib import (
+    validate_input,
+    create_job_name,
+    upload_fasta_to_s3,
+    submit_batch_alphafold_job,
+    get_batch_job_info
+)
+app = Chalice(app_name="lokafold-app")
+
+boto_session = boto3.session.Session()
+region = boto_session.region_name
+batch = boto_session.client("batch", region_name=region)
+
+S3_BUCKET = os.environ["FASTA_BUCKET"]
+
+logging.basicConfig(
+    format="%(asctime)s %(levelname)-s:%(name)s: %(message)s",
+    level=logging.INFO,
+    datefmt="%H:%M:%S",
+    stream=sys.stderr
+)
+logger = logging.getLogger(__name__)
+
+@app.route("/", methods=["POST", "GET"])
```

Comment:
So as we are not using DELETE, PATCH and PUT, can we erase those from the policies role?

Reply:
No... those errors arise even when those methods weren't implemented